### ISSUE TITLE
ReferenceError: c is not defined on Windows

### ISSUE THREAD

**Issue by bisubus** [iccicci/rotating-file-stream#7]
Every several hours app process (debugged in WebStorm) causes Windows dialogue:

> Windows cannot open this file:
> File: tmp-<...>.tmp

When Cancel is pressed on the dialogue, an error is thrown with stack:

```
<...>\node_modules\rotating-file-stream\node_modules\tmp\lib\tmp.js:355
      if (e.errno != -_c.EBADF && e.errno != -c.ENOENT) {
                                              ^
ReferenceError: c is not defined
    at _removeCallback (<...>\node_modules\rotating-file-stream\node_modules\tmp\lib\tmp.js:355:47)
    at _cleanupCallback (<...>\node_modules\rotating-file-stream\node_modules\tmp\lib\tmp.js:409:5)
    at cb (<...>\node_modules\rotating-file-stream\compress.js:55:4)
    at ChildProcess.exithandler (child_process.js:197:7)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
```

The issue is likely related to raszi/node-tmp#102.

**Comment by iccicci** [comment]
This seems to be related with a tmp package problem.
I just released v1.1.1 where I only updated the dependencies.

Please let me know if this was enough to solve the problem.

Thank you,
iCC

**Comment by bisubus** [comment]
Great, thanks. Hope this will help. I will re-open the issue if it won't.
